tailwind.config.js

```diff
@@ -1,12 +1,8 @@
 /** @type {import('tailwindcss').Config} */
 export default {
-  content: [
-    './templates/**/*.twig',
-    './src/**/*.{js,jsx,ts,tsx,svg}',
-  ],
+  content: ['./templates/**/*.twig', './src/**/*.{js,jsx,ts,tsx,svg}'],
```

Comment:
Was this automatic? It’s less readable to me.

Reply:
I wonder if we can adjust prettier... https://www.npmjs.com/package/prettier-plugin-multiline-arrays

Reply:
I'm hesitant to add an extra third-party dependency for something like this.

Prettier does plenty of other things that I view as "not-my-fav". Could end up playing prettier plugin whack-a-mole.